src/theme/colors.ts

```diff
@@ -9,7 +9,7 @@ export interface ThemeColors {
   statusTag: { stopped: string };
   statusBadge: { default: string };
   link: { default: string };
-  node: { background: string; border: string };
+  node: { background: string; border: string; borderSelected: string };
```

Comment:
The changes in this file appear to be left over from your first attempt with the borders. Can you discard these since they aren't used anymore?

Reply:
You're right. I forgot about hose. Deleting...